library/test/src/test/java/com/bumptech/glide/load/data/mediastore/MediaStoreUtilTest.java

```diff
@@ -0,0 +1,25 @@
+package com.bumptech.glide.load.data.mediastore;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import android.net.Uri;
+import android.provider.MediaStore;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
+
+@RunWith(RobolectricTestRunner.class)
+@Config(sdk = 18)
+public class MediaStoreUtilTest {
+
+  @Test
```

Comment:
These are two different tests - could you split them up?

Reply:
Done.